### ISSUE TITLE
Insufficient contrast ratio for highlighted column in DEX visual

### ISSUE THREAD

**Issue by derek1ee** [confluentinc/vscode#290]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### On what operating system are you seeing the problem?

macOS (Apple/arm64)

### VS Code version

Version: 1.93.1 (Universal)
Commit: 38c31bc77e0dd6ae88a4e9cc93428cc27a56ba40
Date: 2024-09-11T17:20:05.685Z
Electron: 30.4.0
ElectronBuildId: 10073054
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Darwin arm64 22.6.0

### Version of Confluent extension

v0.16.3

### To Reproduce

1. Install, signin to CC, and open a message viewer for topic.
2. Search for a message based on some value
3. Observe the DEX visual

### Current vs. Expected behavior

Observe the bar where the filtered message is in is highlighted, but very, very dim.

Expect the highlight to have sufficient contrast to meet WCAG requirements.

![image](https://github.com/user-attachments/assets/cefc95d7-cf6f-4079-bb76-87587d657888)

### Which area(s) are affected? (Select all that apply)

Messages

**Comment by noeldevelops** [comment]
color contrast ratios are hard with varied themes. We may need another pattern. @alexeyraspopov to follow up with design on "contrast/highlight" patterns

**Comment by shouples** [comment]
Closing since this is over a year old - let's reopen this once we get design input with VS Code theme compatibility in mind.
